kubernetes/helm/thirdeye/values.yaml

```diff
@@ -0,0 +1,73 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Default values for ThirdEye Deployment
+
+image:
+  repository: apachepinot/thirdeye
+  tag: latest
+  pullPolicy: IfNotPresent
+
+frontend:
+  enabled: true
+  name: frontend
+  replicaCount: 1
+  port: 1426
+  adminport: 1427
+
+  resources: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  podAnnotations: {}
+
+
+backend:
+  enabled: true
+  name: backend
+  replicaCount: 1
+  port: 1867
+  adminport: 1868
+
+  resources: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  podAnnotations: {}
+
+# Persistence layer configuration
+mysql:
+  mysqlRootPassword: password
```

Comment:
I have kept install.sh extendible to easily override helm parameters. That should give a user some flexibility to feed in secrets. However, this part needs to be resolved in a better way. Since this is a demo setup, I am thinking of starting with this for now as we move to better solutions with k8s secrets, etc.

Reply:
ok let's do this next iteration then